russian/wca-regulations.md

```diff
@@ -509,11 +510,40 @@
         - H1a2) Участник может попросить не разглашать заявленное им число, пока остальные участники не сообщат организаторам их числа кубов. Как только все участники сообщили, сколько кубов они будут пытаться собрать, эта информация становится открытой (например, участник может спрашивать числа других участников до начала попытки).
         - H1a3) До начала попытки, перемешанные головоломки нужно положить на стол в произвольной ориентации, при этом нужно удостовериться, что они полностью закрыты. Головоломки следует разложить в форме, как можно более близкой к квадрату (например, 8 головоломок следует разложить в 2 ряда по 3 головоломки и 1 ряд по 2 головоломки).
     - H1b) Если участник пытается собрать менее 6 кубов, то его лимит времени равен 10 минутам на куб, в противном случае ему даётся час.
         - H1b1) Участник в любое время может дать понять, что попытка окончена. Когда истекает лимит времени, судья сам останавливает попытку и записывает результат; при этом время попытки принимается равным лимиту.
     - H1d) Штрафы за все кубы в попытке накапливаются.
+
+
+## <article-Y><temporary-regulations></article-Y> Статья Y: Временные положения
+
+- Y1) Ниже описаны дополнительные положения, которые замещают собой другие. Данные положения призваны дополнить [Competition Safety Policy](https://www.worldcubeassociation.org/documents/policies/external/Competition%20Safety.pdf).
+- Y2) Определения:
+    - Y2a) Посетитель - любой человек, посещающий соревнование: участник, официальное лицо или гость.
+    - Y2b) Фиксированная рассадка участников - способ проведения соревнования, при котором участники делают свои попытки в данном раунде на одном и том же месте для сборки.
+- Y3) Помимо требований, перечисленных в [Положении 2a](regulations:regulation:2a), к участнику дополнительно могут предъявляться следующие требования, если Делегат WCA считает их необходимыми, а также о них было объявлено заранее:
+    - Y3a) Носить защитную медицинскую маску в месте проведения соревнования.
+    - Y3b) Не иметь контакта с человеком с COVID-19 по крайней мере в течение 14 дней до начала соревнования.
+    - Y3c) Не иметь COVID-19 или испытывать симптомы COVID-19 (определенные [Всмемирной организацией здравоохранения](https://www.who.int/)) по крайней мере в течение 14 дней до начала соревнования.
+    - Y3d) Соблюдать остальные разумные меры безопасности, введённые Всемирной организацией здравоохранения или местными властями.
+- Y4) Всем участникам и официальным лицам следует приносить свои собственные ручки на соревнование.
+- Y5) Судьи могут принести свой собственный секундомер или телефон для измерения времени преинспекции, которые следует проверить Делегату WCA.
+- Y6) Окружающая обстановка:
+    - Y6a) Если Делегат WCA считает это необходимым, в месте проведения соревнования не должно быть зоны для участников.
+    - Y6b) Участникам не следует трогать поверхность стола для сборки, если только они не в процессе выполнения официальной попытки.
```

Comment:
I think it's either a translation error or the official version has changed:
```suggestion
    - Y6b) Участник может накрыть чем-то поверхность стола для сборки, если это не повлияет на работу таймера.
```

Reply:
I was referring to [this PR](https://github.com/thewca/wca-regulations/pull/980) when watching diffs, and Y6b doesn't look like this. What are you referring to?

Reply:
Y6b) Competitors may bring their own cover to place over the competition supplied mat or timer provided the function of the equipment is not affected.

Reply:
According to thewca/wca-regulations#91
- Y6b) Competitors should refrain from touching the surface of the solving station when they are not in the process of an official attempt.

Reply:
I'm sorry, in the last commit, these articles were changed. I looked at an older commit. :)